code/modules/vore/eating/bellymodes_vr.dm

```diff
@@ -744,6 +769,214 @@
 						internal_contents -= P
 		return
 
+///////////////////////////// DM_TRANSFORM_REPLICA_EGG /////////////////////////////
+	if(digest_mode == DM_TRANSFORM_REPLICA_EGG && ishuman(owner))
+		for (var/mob/living/carbon/human/P in internal_contents)
+			if(P.stat)
+				continue
+			if(P.absorbed) //If they're absorbed, don't egg them
+				return
+			var/mob/living/carbon/human/O = owner
+
+			if (O.custom_species)
```

Comment:
I don't think `O.custom_species` will do what you want it to do here. Many people don't have that set. That's the variable for displaying different text on your examine message. That means anyone who didn't set it can't do TFs.